packages/firebase_storage/firebase_storage/test/firebase_storage_test.dart

```diff
@@ -2,24 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.9
-
 import 'package:firebase_core/firebase_core.dart';
 import 'package:firebase_storage/firebase_storage.dart';
 import 'package:firebase_storage_platform_interface/firebase_storage_platform_interface.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
 
 import 'mock.dart';
 
-import 'package:mockito/mockito.dart';
-
 void main() {
   setupFirebaseStorageMocks();
-  /*late*/ FirebaseApp app;
-  /*late*/ FirebaseStorage storage;
-  /*late*/ FirebaseStorage storageSecondary;
-  /*late*/ FirebaseApp secondaryApp;
+  FirebaseApp? app;
```

Comment:
I think we made the decision to stick with using `late` on the tests - cc @rrousselGit

Reply:
Indeed, especially in tests. It leads to better error messages

Reply:
I was under the impression we were switching to nullable as `late` hides any analysis errors. Is it changing back? @rrousselGit @Ehesp

Reply:
Ignore - I think Remi addressed the change back to `late` in this [comment](https://github.com/FirebaseExtended/flutterfire/pull/4753#discussion_r573755991)